src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -84,7 +84,28 @@ public override ExportResult Export(in Batch<LogRecord> batch)
 
                 if (logRecord.State != null)
                 {
-                    this.WriteLine($"{"LogRecord.State:",-RightPaddingLength}{logRecord.State}");
+                    if (logRecord.State is IReadOnlyList<KeyValuePair<string, object>> listKvp)
```

Comment:
we may want to do what OTLP exporter does.
https://github.com/open-telemetry/opentelemetry-dotnet/pull/3186#issuecomment-1104642714

Reply:
Since `ConsoleExporter` is only meant to be used for local testing, I was thinking it's probably a good idea to not impose any option from our end and let the user figure out what works best for them.